src/genvendor/composer.json

```diff
@@ -0,0 +1,78 @@
+{
```

Comment:
Is it required to move `composer.json` from `src/composer.json`?

Reply:
If testing is sorted out then I have alternate plans for it. For now, it must be moved to the build directory. I can refactor the script to use it from the base directory too.